plutus-scb/test/Plutus/SCB/Effects/ContractTest.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE DataKinds        #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs            #-}
+{-# LANGUAGE LambdaCase       #-}
+{-# LANGUAGE TypeOperators    #-}
+module Plutus.SCB.Effects.ContractTest(
+    handleContractTest
+    ) where
+
+import           Control.Monad.Freer
+import           Control.Monad.Freer.Error                     (Error, throwError)
+import           Data.Aeson                                    as JSON
+import           Data.Aeson.Types                              as JSON
+import           Data.Bifunctor                                (Bifunctor (..))
+import           Data.Text                                     (Text)
+import qualified Data.Text                                     as Text
+
+import           Plutus.SCB.Effects.Contract                   (ContractCommand (..), ContractEffect (..))
+import           Plutus.SCB.Types                              (SCBError (..))
+
+import           Language.Plutus.Contract.Resumable            (ResumableError)
+import           Language.Plutus.Contract.Servant              (initialResponse, runUpdate)
+import qualified Language.PlutusTx.Coordination.Contracts.Game as Contracts.Game
+
+-- | A mock/test handler for 'ContractEffect'
+handleContractTest ::
+    (Member (Error SCBError) effs)
+    => Eff (ContractEffect ': effs)
+    ~> Eff effs
+handleContractTest = interpret $ \case
+    InvokeContract (InitContract "game") ->
```

Comment:
Hm, are we just missing the ability to "install" contracts? Seems like we shouldn't have to hardcode them.

Reply:
Ah, it's the same as currently.

Reply:
Yeah it's the same as before, I'm working on improving the install situation atm